pkg/helmreconciler/interfaces.go

```diff
@@ -84,6 +84,18 @@ type RenderingInput interface {
 	GetProcessingOrder(manifests ChartManifestsMap) ([]string, error)
 }
 
+// RenderingInputV2 specifies the details used for rendering charts for v1alpha2 IstioControlPlane.
+type RenderingInputV2 interface {
+	// GetCRtPath returns the path locating the IstioControlPlane CR.
+	GetCRPath() string
+	// GetProcessingOrder is a hook which allows a user to specify the order in which the generated charts
+	// should be applied.  manifests maps chart name to a list of manifests.  Examples of chart names:
+	// istio, istio/charts/security, istio/charts/galley, etc.  Subcharts will have the form:
+	// <main-chart-name>/charts/<subchart-name>
```

Comment:
Does this include CRDS?

Reply:
Yes, that is one of the components. See inputs.go componentDependencies for the dependency tree currently defined - the CRDs are the root of the tree.